Configurations/15-android.conf

```diff
@@ -199,18 +199,22 @@ my %targets = (
         # -march and/or -mfloat-abi flags. NDK defaults to armv5te.
         # Newer NDK versions reportedly require additional -latomic.
         #
-        inherit_from     => [ "android", asm("armv4_asm") ],
+        inherit_from     => [ "android" ],
         bn_ops           => add("RC4_CHAR"),
+        asm_arch         => 'armv4',
+        perlasm_scheme   => "void",
```

Comment:
Since you're changing all this, you should think about documenting ```perlasm_scheme``` and perhaps ```asm_arch```

Reply:
`perlasm_scheme` is already documented, to the best of my knowledge.  I've thoughts on rewriting the documentation in Configurations, and five deeper into perlasm_scheme, but that's for another PR and another day...